### PR TITLE
Add "Greetings and Farewells" transformation

### PR DESCRIPTION
**Pull request by paulozip**
Hello everyone. Congratulations on this awesome project. I think it will be a big contribution to the community.

I created this transformation that **basically creates some text variants and perturbations on words and expressions related to greetings and farewells**. For instance, sentences containing words like "Hi!" or "What's up?" will have these words replaced by similar ones, like "Howdy" or "Good morning". The same behavior will be done for farewells and special cases (e.g. "It's nice to meet you" or "How's it going?").

**The main goal of this transformation is to tackle the robustness of text classification/generation models when faced with perturbations**.

Let me know if you need more details about the implementation.

### CODE REVIEW

File: transformations/greetings_and_farewells/transformation.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    re.compile("have a (great|nice|good) (day|night|week|weekend)", re.IGNORECASE),
+    re.compile("best regards", re.IGNORECASE),
+    re.compile(
+        r"\b(by+e+)+\b", re.IGNORECASE
+    ),  # it matches 'bye', 'bye bye', 'byyyyyyye', 'byeeeee', etc.
+)
+
+
+def greetings_and_farewells(text, seed=0, max_output=1):
+    random.seed(seed)
```
Comment (samsontmr) [inline transformations/greetings_and_farewells/transformation.py:77]: I believe `random.seed()` needs to be called each time before `random.choice()`. I would recommend generating a random integer as a seed for each `random.choice()` call using this seed to increase diversity in the test examples while remaining reproducible.

**Review by samsontmr — CHANGES_REQUESTED** [review]
Thanks for the contribution! This generally looks good. I noticed that you're located in Brazil; it would be great if you could implement this transformation for any other languages you speak!

**Comment by kaustubhdhole** [comment]
Hi @paulozip, thank you for you changes! Here is my overall review:

Build: passes
Interface: Only tagging won't be applicable since the number of tokens are changing in some cases.
Specificity: This change is specific.
Paraphrasing: I think the coverage is very low.
Description: README describes properly.
Evaluating Robustness: No evaluation has been conducted. 
Test cases: 5 have been added.

I find it fine overall, please remove TEXT_TAGGING and if possible, add more variations to increase coverage.

Thanks for your effort!

**Review by samsontmr — APPROVED** [review]
Approving this, the extension to other languages can be done in a separate PR.

**Review by kaustubhdhole — APPROVED** [review]

**Pull request merged**